install.php

```diff
@@ -133,7 +133,7 @@ function ArraySearchRecursive($Needle,$Haystack,$NeedleKey="",$Strict=false,$Pat
 	$usePeople=($result->rowCount()>0 && !ArraySearchRecursive('fac_People',$test))?($result->rowCount()>0 && !ArraySearchRecursive('fac_people',$test))?false:true:true;
 
 	// New install so create a user
-	require_once("customers.inc.php");
+	require_once("classes/People.class.php");
```

Comment:
This would also need to pull in the old user class, not just people.  The check on line 133 is determining which class to use and in the case of upgrades it needs the old class.